packages/cascara/src/ui/Table/Table.test.js

```diff
@@ -57,86 +57,54 @@ describe('Table', () => {
       ],
     };
 
-    const dataConfig = {
-      display: [
-        {
-          attribute: 'active',
-          isEditable: true,
-          label: 'Active',
-          module: 'checkbox',
-        },
-        {
-          attribute: 'eid',
-          isEditable: false,
-          label: 'ID',
-          module: 'text',
-        },
-        {
-          attribute: 'email',
-          isEditable: true,
-          label: 'Email',
-          module: 'email',
-        },
-        {
-          attribute: 'country',
-          isEditable: true,
-          label: 'Country',
-          module: 'select',
-          options: [
-            {
-              label: 'Argentina',
-              value: 'Argentina',
-            },
-            {
-              label: 'Brazil',
-              value: 'Brazil',
-            },
-            {
-              label: 'USA',
-              value: 'USA',
-            },
-          ],
-        },
-        {
-          attribute: 'employeeNumber',
-          isEditable: true,
-          label: 'Employee Number',
-          module: 'number',
-        },
-        {
-          attribute: 'fullName',
-          isEditable: true,
-          label: 'Full Name',
-          module: 'text',
-        },
-        {
-          attribute: 'homePhone',
-          isEditable: true,
-          label: 'Home Phone',
-          module: 'text',
-        },
-        {
-          attribute: 'officePhone',
-          isEditable: true,
-          label: 'Office Phone',
-          module: 'text',
-        },
-        {
-          attribute: 'title',
-          isEditable: true,
-          label: 'Title',
-          module: 'text',
-        },
-      ],
-    };
+    const dataDisplay = [
+      {
+        attribute: 'active',
+        isEditable: true,
+        label: 'Active',
+        module: 'checkbox',
+      },
+      {
+        attribute: 'eid',
+        isEditable: false,
+        label: 'ID',
+        module: 'text',
+      },
+      {
+        attribute: 'email',
+        isEditable: true,
+        label: 'Email',
+        module: 'email',
+      },
+      {
+        attribute: 'country',
+        isEditable: true,
+        label: 'Country',
+        module: 'select',
+        options: [
+          {
+            label: 'Argentina',
+            value: 'Argentina',
+          },
+          {
+            label: 'Brazil',
+            value: 'Brazil',
+          },
+          {
+            label: 'USA',
+            value: 'USA',
+          },
+        ],
+      },
+    ];
 
     test('snapshot test', () => {
```

Comment:
@lordpixel it looks like you did a test separating out the snapshot from the rest of this test and that did decrease the overall runtime and we went with reducing the test data size to make the tests run faster instead because that gave us a larger margin of improvement.

Did we evaluate what doing _both_ of these things does to the time? Do we get slightly better performance still from both separating tests and also reducing the data test size?

Reply:
Yes we did, with both approaches, we observed is an increase in running time, not a decrease. It's the last screenshot I posted.